kolibri/core/discovery/utils/filesystem/base.py

```diff
@@ -0,0 +1,78 @@
+import logging
```

Comment:
Yeah, initially I was planning to make it object-oriented, and have a base class in here that the OS-specific libs would import, but I ended up going a different direction.

I'm hesitant to use `api.py`, since we use that for DRF stuff.

Yeah, I was thinking to put it in __init__.py, but I kept hearing @benjaoming in the back of my head saying to keep those files empty... But I think the main concern was around circular imports; if we only import functions from within the module itself, it could be safe. But might hide them away a bit...